src/test/resources/schema/ome_zarr_0.4.schema

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
Ideally you wouldn't copy the scheme, but would download it from `"https://ngff.openmicroscopy.org/0.4/schemas/image.schema"` again to pull in potential updates. But for now we can also just copy it to keep this simple.

Reply:
Good point - let me see if there's an easy fix for this.

Reply:
Ok - wasn't too hard to fix this! I download the schema directly from github now